### PR TITLE
Implementation pages add default description combine settings

### PR DESCRIPTION
**Pull request by aghimpu**

### CODE REVIEW

File: src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js [hunk 1]
```diff
@@ -26,6 +26,17 @@ const ImplementationCmsPageEditComponent = function (
...
+    $ctrl.descriptionPositions = [{
+        value: 'replace',
+        name: 'Overwrite default page content',
+    }, {
+        value: 'before',
+        name: 'Show before default page content',
+    }, {
+        value: 'after',
+        name: 'Show after default page content',
+    }];
```
Comment (lexlog) [inline src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js:38]: words above need translation, @Dominique2002 could you suggest something for this 3 sentences (check them in full file view or diff)?

**Comment by lexlog** [comment]
@lexlog write a comment what is not working on dev

File: src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js (hunk 1 above)
Comment (Dominique2002) [reply src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js:38]: I think these sentences would be good translations

'Standaard pagina-inhoud overschrijven' (overwrite)

'Voor standaard pagina-inhoud tonen' (before)

'Na standaard pagina-inhoud tonen' (after)

Comment (lexlog) [reply src/sources/forus-platform/js/angular-1/components/ImplementationCmsPageEditComponent.js:38]: @Dominique2002 thank you!

**Review by lexlog — APPROVED** [review]

**Pull request merged**